apps/balancer-tools/src/app/apr/pool/(components)/PoolTokensTable.tsx

```diff
@@ -0,0 +1,116 @@
+"use client";
+import { ExternalLinkIcon } from "@radix-ui/react-icons";
+import Image from "next/image";
+
+import Table from "#/components/Table";
+
+import { PoolTokens } from "../../api/route";
+
+export default function PoolTokensTable({
+  poolTokensStats,
+}: {
+  poolTokensStats: PoolTokens[];
+}) {
+  const tokenBalanceUSD = (balance: string) =>
+    parseFloat(balance).toLocaleString("en-US", {
+      minimumFractionDigits: 0,
+      maximumFractionDigits: 0,
+    });
+  return (
+    <>
+      <Table color="blue" shade={"darkWithBorder"}>
+        <Table.HeaderRow>
+          <Table.HeaderCell>Name</Table.HeaderCell>
+          {poolTokensStats[0].weight && (
+            <Table.HeaderCell classNames="w-1/6 text-end p-4">
+              Weight
+            </Table.HeaderCell>
+          )}
+          {poolTokensStats[0].balance && (
+            <Table.HeaderCell classNames="w-1/6 text-end p-4">
+              Balance
+            </Table.HeaderCell>
+          )}
+          {poolTokensStats[0].price && poolTokensStats[0].balance && (
+            <Table.HeaderCell classNames="w-1/6 text-end p-4">
+              Value
+            </Table.HeaderCell>
+          )}
+          {poolTokensStats[0].price && poolTokensStats[0].balance && (
+            <Table.HeaderCell classNames="w-1/6 text-end p-4">
+              Token %
+            </Table.HeaderCell>
+          )}
+        </Table.HeaderRow>
+        <Table.Body>
+          {poolTokensStats.map((token) => (
+            <Table.BodyRow classNames="hover:bg-blue4 hover:cursor-pointer duration-500">
+              <Table.BodyCellLink
+                href={`https://etherscan.io/address/${token.address}`}
```

Comment:
tokens won't all be in etherscan -- remember this is multichain. I think we have a canonical list of chain scanners somewhere